learn/wavemaker-release-notes/v10-14-0.md

```diff
@@ -0,0 +1,106 @@
+---
+title: "WaveMaker 10.14.0 - Release date: 1 May 2022"
+id: ""
+sidebar_label: "Release v10.14.0"
+---
+*WaveMaker released another version update with new features and bug fixes.*
+
+## New Features
+---
+
+### SSL for Deployed App URLs
+
+Introducing SSL for Deployed Apps to [Demo Cloud](/learn/app-development/deployment/pipelines-phases#deployment-environment), allowing you to access apps with the secure protocol HTTPS. To support this, we made changes to the App URL pattern. Below are the changes.
+
+**WMO Personal workspace**
+
+Old: `http://p8888888n45.cloud.wavemakeronline.com/TestApp`  
+New: `http://cloud.wavemakeronline.com/pk8888888n45/TestApp`  
+
+:::note
+In the URL, `pk8888888n45` is a Project key.
+:::
+
+**Teams and Enterprise**
+
+Old: `http://wmshowcase.cloud.wavemakeronline.com/TestApp`  
+New: `http://cloud.wavemakeronline.com/wmshowcase/TestApp`  
+
+:::note
+`wmshowcase` is the WaveMaker Online team's name and WaveMaker Enterprise's name in the URL.
+:::
+
+For more information, see [Demo Cloud](/learn/app-development/deployment/pipelines-phases#deployment-environment).
+
+## Improvements
+---
+
+- Docker-compose accept env variables with character `_`.
+
+
+## Bug fixes
+---
+
+- Fixed an issue related to on-demand pagination in Datatable, which disappears when all the rows are expanded and sorted by any displayed column.
+- Fixed an issue related to Datatable `Header Onclick` event not triggering.
+- Fixed an issue related to Veracode Scan in AppRuntimeController.
+- Fixed an issue related to SOAP service is not working in preview showing NoClassDefFoundError.
+- Fixed an alignment issue of Textarea's helptext when used in Form. 
+- Fixed an issue related to Oauth WebService Variable not triggering Success callback in the first call.
+
+### Team Portal
```

Comment:
Not required this. these are fixes during development. regresssion kind of things..
Nothing to mention.